### ISSUE TITLE
How to calculate classification accuracy

### ISSUE THREAD

**Issue by whiterAutumn** [lightly-ai/lightly#1610]
Is there a brief testing script that allows us to validate the classification accuracy of the model while training?looking forward to your reply.

**Comment by guarin** [comment]
Hi! The simplest approach for validating accuracy during training is to use an online linear classifier. If you are using PytorchLightning you can find an example here:

https://github.com/lightly-ai/lightly/blob/8b0163843079f0c5cea41031349c9c97dff9635a/benchmarks/imagenet/resnet50/barlowtwins.py#L31

And here is how it is used during training and validation:
https://github.com/lightly-ai/lightly/blob/8b0163843079f0c5cea41031349c9c97dff9635a/benchmarks/imagenet/resnet50/barlowtwins.py#L51-L55

https://github.com/lightly-ai/lightly/blob/8b0163843079f0c5cea41031349c9c97dff9635a/benchmarks/imagenet/resnet50/barlowtwins.py#L62-L66

**Comment by whiterAutumn** [comment]
thanks a lot , and  if there is a PyTorch version available, not PytorchLightning?

**Comment by guarin** [comment]
We don't have an example right now but you should be able to use the `OnlineLinearClassifier` also with PyTorch. If you don't want the pytorch lightning dependency you can copy the code and replace the `LightningModule` superclass with `torch.nn.Module`.

**Comment by whiterAutumn** [comment]
thank you sincerely . I will try it.

**Comment by guarin** [comment]
Will close this for now, please reopen if you have more questions.
